server/webapp/WEB-INF/rails.new/app/controllers/agents_controller.rb

```diff
@@ -38,6 +38,13 @@ class AgentsController < ApplicationController
   layout "application"
   prepend_before_filter :set_tab_name
 
+  def check_feature_toggle
+    if Toggles.isToggleOn(Toggles.AGENTS_SINGLE_PAGE_APP)
```

Comment:
This can just be `Toggles.isToggleOn(Toggles.AGENTS_SINGLE_PAGE_APP)`, also the convention for methods that return a boolean is to add a `?` — so `is_feature_on?`